Sources/SwiftGenKit/Parsers/FontsParser/CTFont.swift

```diff
@@ -0,0 +1,46 @@
+//
+// SwiftGenKit
+// Copyright (c) 2017 SwiftGen
+// Created by Derek Ostrander on 3/7/16.
+// MIT License
+//
+
+import AppKit.NSFont
+import Foundation
+import PathKit
+
+extension PathKit.Path {
+  /// Returns the Path relative to a parent directory.
+  /// If the argument passed as parent isn't a prefix of `self`, returns `nil`.
+  ///
+  /// - Parameter parent: The parent Path to get the relative path against
+  /// - Returns: The relative Path, or nil if parent was not a parent dir of self
+  func relative(to parent: Path) -> Path? {
+    let parentComponents = parent.absolute().components
```

Comment:
Why is this method, which is quite applicable to any path and could even one day be moved in PathKit directly, is only located in CTFont?

Reply:
Same as above, it was a free function and I wanted to compartmentalise it a bit. Move it into CTFont?